addons/sourcemod/scripting/mgemod.sp

```diff
@@ -45,180 +46,202 @@
 
 // Midair
-new g_iMidairHP;
+int g_iMidairHP;
 
 // Debug log
-new String:g_sLogFile[PLATFORM_MAX_PATH];
+char g_sLogFile[PLATFORM_MAX_PATH];
 
 // Endif
-new Float:g_fRocketForceX,
-	Float:g_fRocketForceY,
-	Float:g_fRocketForceZ;
+float 
+	g_fRocketForceX,
+	g_fRocketForceY,
+	g_fRocketForceZ;
 
 // Bball
-new String:g_sBBallParticleRed[64],
-	String:g_sBBallParticleBlue[64];
-
-static const String:stockSounds[][]= // Sounds that do not need to be downloaded.
-{						
-						"vo/intel_teamcaptured.wav",
-						"vo/intel_teamdropped.wav",
-						"vo/intel_teamstolen.wav",
-						"vo/intel_enemycaptured.wav",
-						"vo/intel_enemydropped.wav",
-						"vo/intel_enemystolen.wav",
-						"vo/announcer_ends_5sec.wav",
-						"vo/announcer_ends_4sec.wav",
-						"vo/announcer_ends_3sec.wav",
-						"vo/announcer_ends_2sec.wav",
-						"vo/announcer_ends_1sec.wav",
-						"vo/announcer_ends_10sec.wav",
-						"vo/announcer_control_point_warning.wav",
-						"vo/announcer_control_point_warning2.wav",
-						"vo/announcer_control_point_warning3.wav",
-						"vo/announcer_overtime.wav",
-						"vo/announcer_overtime2.wav",
-						"vo/announcer_overtime3.wav",
-						"vo/announcer_overtime4.wav",
-						"vo/announcer_we_captured_control.wav",
-						"vo/announcer_we_lost_control.wav",
-						"items/spawn_item.wav",
-						"vo/announcer_victory.wav",
-						"vo/announcer_you_failed.wav"
+char 
+	g_sBBallParticleRed[64],
+	g_sBBallParticleBlue[64];
+
+static const char stockSounds[][] =  // Sounds that do not need to be downloaded.
+{
+	"vo/intel_teamcaptured.wav", 
+	"vo/intel_teamdropped.wav", 
+	"vo/intel_teamstolen.wav", 
+	"vo/intel_enemycaptured.wav", 
+	"vo/intel_enemydropped.wav", 
+	"vo/intel_enemystolen.wav", 
+	"vo/announcer_ends_5sec.wav", 
+	"vo/announcer_ends_4sec.wav", 
+	"vo/announcer_ends_3sec.wav", 
+	"vo/announcer_ends_2sec.wav", 
+	"vo/announcer_ends_1sec.wav", 
+	"vo/announcer_ends_10sec.wav", 
+	"vo/announcer_control_point_warning.wav", 
+	"vo/announcer_control_point_warning2.wav", 
+	"vo/announcer_control_point_warning3.wav", 
+	"vo/announcer_overtime.wav", 
+	"vo/announcer_overtime2.wav", 
+	"vo/announcer_overtime3.wav", 
+	"vo/announcer_overtime4.wav", 
+	"vo/announcer_we_captured_control.wav", 
+	"vo/announcer_we_lost_control.wav", 
+	"items/spawn_item.wav", 
+	"vo/announcer_victory.wav", 
+	"vo/announcer_you_failed.wav"
 };
 
-public Plugin:myinfo =
+public Plugin myinfo =
 {
-  name = "MGEMod",
-  author = "Lange & Cprice; based on kAmmomod by Krolus - maintained by sappho.io",
-  description = "Duel mod with realistic game situations.",
-  version = PL_VERSION,
-  url = "https://github.com/Langeh/MGEMod, http://steamcommunity.com/id/langeh"
+	name = "MGEMod",
+	author = "Lange & Cprice; based on kAmmomod by Krolus - maintained by sappho.io // ampere",
+	description = "Duel mod with realistic game situations.",
+	version = PL_VERSION,
+	url = "https://github.com/Langeh/MGEMod, http://steamcommunity.com/id/langeh"
```

Comment:
update urls to https://github.com/sapphonie/MGEMod and https://sappho.io altho this is more of a note to self than anything